src/mzcloud-cli/src/bin/mzcloud.rs

```diff
@@ -145,7 +145,7 @@ enum DeploymentsCommand {
         disable_user_indexes: Option<bool>,
 
         /// Extra arguments to provide to materialized.
-        #[clap(long)]
+        #[clap(long, allow_hyphen_values = true, multiple_values = true)]
```

Comment:
Hrm. This seems like a problematic option to set.

<img width="962" alt="image" src="https://user-images.githubusercontent.com/882976/148470023-6b3f9a2c-a0c4-4b1f-87a2-309299ee548b.png">

It seems like it would be safer to use `multiple_occurrences` instead? You'd have to type a lot but it's only for internal use anyway. 🤷🏽‍♂️ 

```
--materialized-extra-args --foo --materialized-extra-args --bar
```

Reply:
Yeah, I thought about that. This way works fine, but `--materialized-extra-args` has to be the last argument, and you can't use `--` to separate positional args.

If you prefer the `multiple_occurrences` way, that's fine. It just seemed annoying to have to repeat such a long argument. Maybe we should shorten it to something more manageable.

Perhaps there is a way to make clap accept it as a single long string, but still parse that string into a `Vec` for us.

Reply:
It's not like users would ever have access to this so I guess it doesn't much matter.